src/settings/routes/ExternalDataSourcesSettingsRoute/ExternalDataSourcesSettingsRoute.js

```diff
@@ -50,10 +50,15 @@ const ExternalDataSourcesSettingsRoute = () => {
 
   const { mutateAsync: putExternalKB } = useMutation(
     ['ERM', 'KBs', 'PUT'],
-    (payload) => {
-      ky.put(`${KBS_ENDPOINT}/${payload.id}`, { json: payload }).json().then(() => {
+    async (payload) => {
+      try {
+        await ky.put(`${KBS_ENDPOINT}/${payload.id}`, { json: payload }).json();
```

Comment:
Can this please be changed back to `payload => ky.put(...).then()` but then including a `.catch()` block for the caught stuff?

Reply:
See our slack conversation, will not change that